docs/ivys.js

```diff
@@ -54,6 +76,8 @@ var emptyProductState = function (params) {
         tags: {},
         // Collection name
         collection: params.collection
+        // Should I also include the image in here? 
```

Comment:
You should include all of the attributes that you expect to need. I think including a link to the image and/or a thumbnail is useful here, otherwise you're just going to have to look it up every time you want to manipulate it.